### ISSUE TITLE
Add option to anonymize IPs using Google Analytics

### ISSUE THREAD

**Issue by rostockahoi** [khalwat/seomatic#188]
At first: great plugin, I love it.

German authorities force website owners to do a list of things when using Google Analytics including IP anonymization and an opt-out for tracking.

Let's stick to anonymization. If it is already built-in in SEOmatic I'm just too dumb to see it. If it is not it would be great as a new feature. I suggest it as follows:

Basically it's just a single line of code to add to the GA init script in the `<head>` tag.

``` javascript
<script type="text/javascript">
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-XXXXXXXX', 'auto');
    ga('set', 'anonymizeIp', true);  // <-- we need this line at exactly this position before send pageview
    ga('send', 'pageview');
</script>
```

Regarding the backend it could look something like this:

![auswahl_015](https://cloud.githubusercontent.com/assets/16897125/22330829/1554ce2c-e3c0-11e6-93a7-0a5133a75ed4.png)

**Comment by rostockahoi** [comment]
What I forgot: I really would implement it by myself and create a PR but my skill in Craft plugin dev is nothing to be proud of at this time. Maybe in the future...

**Comment by khalwat** [comment]
This shouldn't be too bad to do.

**Comment by khalwat** [comment]
Implemented in 1.1.42: https://github.com/nystudio107/seomatic/releases/tag/1.1.42

Sorry I didn't see your PR before I did this :(

![google chromescreensnapz345](https://cloud.githubusercontent.com/assets/7570798/22866369/81e5e9c8-f143-11e6-9ea1-7a4a40c15e2d.png)

![google chromescreensnapz347](https://cloud.githubusercontent.com/assets/7570798/22866371/863c8a90-f143-11e6-9801-1b1bb6f679fa.png)
